bindings/python/convert.py

```diff
@@ -31,6 +44,8 @@ def convert_multi(model_id):
         commit_message="Adding `safetensors` variant of this model",
         create_pr=True,
     )
+    for local in local_filenames:
+        os.remove(local)
```

Comment:
Looks like the kind of things that should go in a `finally` and the rest should be in a `try` block, to avoid leaving the user with files in case of error during the script.

Reply:
Updated. I didn't meant to make this the most bullet proof but you're right so I added this.